src/bot/say-hello.js

```diff
@@ -7,24 +7,23 @@ export function sayHello() {
     const messageWords = text.toLowerCase().split(' ')
 
     if (!messageWords.some(v => hellos.includes(v))) {
-      return { ...messageContext }
+      return
     }
 
     const mentionSomeone = messageWords.filter(w => w.startsWith('@')).length > 0
     const mentionSanpe = mentionSomeone && messageWords.includes(`@${process.env.BOT_USERNAME.toLowerCase()}`)
 
     if (mentionSomeone && !mentionSanpe) {
       // mention
-      return { ...messageContext }
+      return
     }
 
     if (helloUserMemory.has(username) && !mentionSanpe) {
-      return { ...messageContext }
+      return
     }
 
     helloUserMemory.add(username)
     return {
```

Comment:
I think it should set fulfilled at true at this point

Reply:
`fulfilled` is by default equal to `true`. See https://github.com/mindsers/sanpe-bot/pull/17/files#diff-f4fd88ee249fe934c063de2e756fd76d0a610ab3ad0068edb1dbbe870344b387R26